src/plugins/xfconf/README.md

```diff
@@ -0,0 +1,51 @@
+- infos = Information about the xfconf plugin is in keys below
+- infos/author = Richard Stöckl <e11908080@student.tuwien.ac.at>
+- infos/licence = BSD
+- infos/needs =
+- infos/provides = storage/xfconf
+- infos/recommends =
+- infos/placements = postgetstorage presetstorage
+- infos/status = maintained libc configurable experimental unfinished concept limited memleak
+- infos/metadata =
+- infos/description = storage plugin for xfconf
+
+## Introduction
+
+This is a storage plugin to mount the xfconf configuration settings.
+
+## Dependencies
+
+xfconf from the XFCE project
+
+## Usage
+
+The usage is identical to most storage plugins except that the channel option during mount must be defined in order to tell xfconf which channel to mount.
```

Comment:
Please give information what channels are, how to list which channels are available etc. Does it maybe make sense to use xfwm4 as default?

Reply:
I added the information about channels in the commits 487ecceb064c9a3af9a9a2af582363165d413ac7 and e18c08c53b8da4e713192f758d8c487df0f17616. I did not even thought about listing channels, good point! I implemented that in the commit a5991352a15906028735122d6c1b4d932707140b.

I do not think that it makes sense to set `xfwm4` as the default channel. This would mean that the mount operation without a channel provided would always result into mounting the configuration of xfwm. Xfwm does not mean anything special to xfconf, it was just used in the examples.